lib/geordi/commands/create_database_yml.rb

```diff
@@ -6,10 +6,12 @@ def create_database_yml
   if File.exists?(sample_yml) and not File.exists?(real_yml)
     announce 'Creating ' + real_yml
 
-    print 'Please enter your DB password: '
+    sample = File.read(sample_yml)
+    adapter = sample.match(/adapter: (\w+)\n/).captures.first
+
+    print "Please enter your DB password for #{adapter}: "
```

Comment:
Please use this output:
```suggestion
    print "Please enter your #{adapter} password: "
```